src/main/java/frc/trigon/robot/constants/CommandConstants.java

```diff
@@ -38,6 +39,10 @@ public class CommandConstants {
             ),
             STATIC_WHITE_LED_COLOR_COMMAND = LEDStripCommands.getStaticColorCommand(Color.white, LEDStripConstants.LED_STRIPS);
 
+    public static double calculateDriveStickAxisValue(double axisValue) {
+        return axisValue / OperatorConstants.STICKS_SPEED_DIVIDER / calculateShiftModeValue(MINIMUM_TRANSLATION_SHIFT_POWER);
+    }
```

Comment:
add this for angle axis value as well, and implement this for all the other drive commands like in the 2024 code. 
We need to add this to the template at some point